tests/utils.go

```diff
@@ -3004,19 +3004,18 @@ func LoggedInAlpineExpecter(vmi *v1.VirtualMachineInstance) (expect.Expecter, er
 	}
 
 	b := append([]expect.Batcher{
-		&expect.BSnd{S: "\n"},
 		&expect.BSnd{S: "\n"},
 		&expect.BExp{R: "localhost login:"},
 		&expect.BSnd{S: "root\n"},
-		&expect.BExp{R: "localhost:~\\#"}})
-	res, err := expecter.ExpectBatch(b, 180*time.Second)
+		&expect.BExp{R: "localhost:~#"}})
```

Comment:
can you explain why this change is needed?

Reply:
`#` is not a special character and therefore does not needs escaping. so this double slash is redundant.
In fact, a new [commit](https://github.com/kubevirt/kubevirt/pull/3883/commits/fdbf06fc96e75e8dc3d3305706bcd518918c272d) was recently merged which generalizes the prompt  experssion.
so I'll change it to use that one instead of  `"localhost:~#"`